src/test/ui/impl-trait/must_outlive_least_region_or_bound.nll.stderr

```diff
@@ -26,7 +26,34 @@ LL | fn explicit<'a>(x: &'a i32) -> impl Copy + 'a { x }
    |                                ^^^^^^^^^^^^^^
 
 error: lifetime may not live long enough
-  --> $DIR/must_outlive_least_region_or_bound.rs:12:69
+  --> $DIR/must_outlive_least_region_or_bound.rs:9:46
+   |
+LL | fn elided2(x: &i32) -> impl Copy + 'static { x }
+   |               -                              ^ returning this value requires that `'1` must outlive `'static`
+   |               |
+   |               let's call the lifetime of this reference `'1`
+   |
+   = help: consider replacing `'1` with `'static`
+
+error: lifetime may not live long enough
+  --> $DIR/must_outlive_least_region_or_bound.rs:12:55
+   |
+LL | fn explicit2<'a>(x: &'a i32) -> impl Copy + 'static { x }
+   |              -- lifetime `'a` defined here            ^ returning this value requires that `'a` must outlive `'static`
+   |
+   = help: consider replacing `'a` with `'static`
+   = help: consider replacing `'a` with `'static`
+
+error[E0621]: explicit lifetime required in the type of `x`
+  --> $DIR/must_outlive_least_region_or_bound.rs:15:41
+   |
+LL | fn foo<'a>(x: &i32) -> impl Copy + 'a { x }
+   |               ----                      ^ lifetime `'a` required
+   |               |
+   |               help: add explicit lifetime `'a` to the type of `x`: `&'a i32`
```

Comment:
I think something as simple as:
```suggestion
   |               help: add an explicit lifetime: `&'a i32`
```
would do; it's easier to read.

Reply:
The explicit lifetime already exists, so saying `an` would imply the _creation_ of a new lifetime (that's how it'd read to me). Removing the lifetime name and the description of `x`'s type removes some jargon, but I think there's a case to be made about this being a good moment to _teach_ that jargon.

Reply:
That sounds reasonable. What do you think about altering slightly to `help: add the explicit lifetime […]`? That reads slightly better to me, but is a little longer.